driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/OperationExecutorImpl.java

```diff
@@ -42,10 +47,10 @@
 
 class OperationExecutorImpl implements OperationExecutor {
```

Comment:
Looks like OperationExecutor is still in driver-core.  It could be pulled up as well.  There is one test implementation of it in core but it looks to be unused.

Reply:
Will look to lifting it.

Reply:
Done